nautobot/core/settings.py

```diff
@@ -1146,7 +1147,10 @@ def silk_request_logging_intercept_logic(request):
 KUBERNETES_DEFAULT_SERVICE_ADDRESS = os.getenv("NAUTOBOT_KUBERNETES_DEFAULT_SERVICE_ADDRESS", "")
 
 # A dictionary that stores the kubernetes pod manifest used to create a job pod in the kubernetes cluster
-KUBERNETES_JOB_MANIFEST = {}
+try:
+    KUBERNETES_JOB_MANIFEST = json.loads( os.getenv("NAUTOBOT_KUBERNETES_JOB_MANIFEST", "{}"))
```

Comment:
Please also update the `nautobot/core/settings.yaml` entry for `KUBERNETES_JOB_MANIFEST` to indicate support for this environment variable.